src/main/java/seedu/duke/commands/Command.java

```diff
@@ -0,0 +1,35 @@
+package seedu.duke.commands;
+
+import seedu.duke.trackables.Task;
+
+import java.util.List;
+
+public abstract class Command {
+
+    static final String HORIZONTAL_LINE = "______________________________"
+            + "______________________________";
+
+    public void execute(List<Task> tasks) {
+
+    }
+
+    /**
+     * Prints out the message.
+     * @param messages Array of messages to show.
+     */
+    protected void echo(String[] messages) {
```

Comment:
Maybe you want to separate the logic of printing, handling database into more classes such as Storage and Parser? This would reduce the responsibility of the Command classes and may aid in coding or making changes when doing further extensions of the program.